xml/System.Diagnostics/ActivityTraceId.xml

```diff
@@ -189,9 +189,9 @@ The characters in <paramref name="idData" /> are not all lower-case hexadecimal
         <Parameter Name="traceId" Type="System.Diagnostics.ActivityTraceId" />
       </Parameters>
       <Docs>
-        <param name="traceId">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="traceId">The <see cref="T:System.Diagnostics.Tracing.ActivityTraceId" /> to compare to.</param>
+        <summary>Compare <see langword="this" /> with another instance of <see cref="T:System.Diagnostics.Tracing.ActivityTraceId" />.</summary>
+        <returns><see langword="true" /> if <paramref name="traceId" /> has the same hex value as <see langword="this" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if <paramref name="traceId" /> has the same hex value as the current instance; otherwise, <see langword="false" />.</returns>
```